config/config-api/src/com/thoughtworks/go/domain/UnitTestReportGenerator.java

```diff
@@ -151,18 +153,12 @@ private void pumpFileContentIfValid(PrintStream out, File testFile) throws IOExc
     }
 
     private void pumpFileContent(File file, PrintStream out) throws IOException {
-        BufferedReader bufferedReader = null;
         try {
-            bufferedReader = new BufferedReader(new FileReader(file));
-            String line = bufferedReader.readLine();
-            if (!line.contains("<?xml")) { // skip prolog
-                out.println(line);
-            }
-            while ((line = bufferedReader.readLine()) != null) {
-                out.println(line);
-            }
-        } finally {
-            IOUtils.closeQuietly(bufferedReader);
+            String content = FileUtil.readToEnd(new BOMInputStream(new FileInputStream(file)));
```

Comment:
I looked at the implementation of `FileUtil.readToEnd` and it does not seem like we're closing the `InputStream`, so we may be leaking an inputstream.

The rest looks OK.